packages/blocks/src/page-block/default/selection-manager/default-selection-manager.ts

```diff
@@ -309,6 +310,14 @@ export class DefaultSelectionManager {
     showFormatQuickBarByDoubleClick(e, this.page, this.container, this.state);
   };
 
+  private _onContainerTripleClick = (e: SelectionEvent) => {
```

Comment:
@lawvs You can add the logic of all the text  of the block in this method.

Reply:
It mainly shows the logic of format-quick-bar and whether the content is correct when copying.